config/initializers/vault.rb

```diff
@@ -1,4 +1,9 @@
-VAULT_ENABLED = ENV.fetch('SECRET_STORAGE_BACKEND', false)
+# some hackery to get this loaded up when testing
+VAULT_ENABLED = ENV.fetch('SECRET_STORAGE_BACKEND', Rails.env.test? ? "SecretStorage::HashicorpVault" : false)
```

Comment:
add `SECRET_STORAGE_BACKEND=XYZ` into `.env.test` ... but either way we are only testing one ... so preferably run the secret storage backend with both ...